### PR TITLE
fix LZMA archive directory structure

### PR DESCRIPTION
**Pull request by anurse**
/cc @livarcocc - Since these changes are mostly in the code that came from your repo ;).

### CODE REVIEW

File: src/dotnet-archive/dotnet-archive.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <OutputType>Exe</OutputType>
     <EnableApiCheck>false</EnableApiCheck>
-    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
```
Comment (JunTaoLuo) [inline src/dotnet-archive/dotnet-archive.csproj]: Any reason this isn't win-x64? We are using the updated consolidated RIDs elsewhere in our build.

Comment (JunTaoLuo) [inline src/dotnet-archive/dotnet-archive.csproj]: The only issue with this is some issue with sqlclient cc @kichalla has more details.

Comment (anurse) [inline src/dotnet-archive/dotnet-archive.csproj]: For this package it's fine. This is only temporary (and it isn't built by our CI, I just manually build it). @livarcocc is creating a separate repo for the archiver so we can share the code properly.

**Comment by anurse** [comment]
@livarcocc - I don't think you'll need to update your extraction code, but I would double check it to verify everything is OK. As a result of this change, the Destination path during extraction for each file will always use `/` as a separator. The only platform at risk is Windows but Windows **does** explicitly allow `/` as an "alternative" directory separator so it should be fine. It was fine in my testing, but I just want to call that out.

When we have a shared Archiver repo, we can make sure the code is unified.

**Comment by livarcocc** [comment]
I picked up the new lzma and CI is passing.

**Pull request merged**